tests/dragonfly/utility.py

```diff
@@ -141,13 +141,26 @@ class ValueType(Enum):
 class CommandGenerator:
     """Class for generating complex command sequences"""
 
-    def __init__(self, target_keys, val_size, batch_size, max_multikey, unsupported_types=[]):
+    def __init__(
+        self,
+        target_keys,
+        val_size,
+        huge_val_count,
+        huge_val_size,
+        batch_size,
+        max_multikey,
+        unsupported_types=[],
+    ):
         self.key_cnt_target = target_keys
         self.val_size = val_size
         self.batch_size = min(batch_size, target_keys)
         self.max_multikey = max_multikey
         self.unsupported_types = unsupported_types
 
+        # Generate sorted list of random samples in target_keys range
+        self.huge_val_sample = sorted(random.sample(range(0, target_keys), huge_val_count))
```

Comment:
how about using 
sorted(random.sample(range(target_keys), huge_val_count))
is more memory efficient relevant if target_keys is very big